<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>745</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>941</width>
    <height>745</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>941</width>
    <height>745</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btn_save_all">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del_raw">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除左侧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del_rgba8888">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>310</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除左侧</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_raw_cnt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>总数 : 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rgba8888_cnt">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>总数 : 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
